packaging/suse/trento-agent.spec

```diff
@@ -49,7 +49,7 @@ Trento agents are client-side processes responsible for the automatic discovery
 %define shortname agent
 
 %build
-VERSION=%{version} INSTALLATIONSOURCE=Suse make build
+VERSION=%{version} INSTALLATIONSOURCE=Suse make build BUILD_OUTPUT="./trento-agent"
```

Comment:
nitpick: I would put `BUILD_OPTION` after `INSTALLATION_SOURCE` and before `make build`, so we have `envs command` structure